### PR TITLE
added auto reload air config file and .env.local file setup

### PR DESCRIPTION
**Pull request by arghyaiitb**
- Added `.env.local` file for local setup
- Added air config setup
- Added dockerfile autoreloading with air

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,6 @@
-FROM golang:1.14
-WORKDIR /app
-COPY go.mod go.sum ./
+FROM golang:1.14.2-alpine3.11
+WORKDIR /usr/src/app
+COPY . /usr/src/app
```
Comment (dmonark) [inline Dockerfile]: After workdir we don’t have to specify path for copy "." will work

`COPY . .`

ref - https://docs.docker.com/engine/reference/builder/#workdir

**Review by dmonark — CHANGES_REQUESTED** [review]
Dockerfile and docker-compose both are in dev config so I think we should dev in the name of file so i doesn’t clash with prod file

**Comment by arghyaiitb** [comment]
> Dockerfile and docker-compose both are in dev config so I think we should dev in the name of file so i doesn’t clash with prod file

When we will move to prod/staging lets configure it then. For now lets go ahead with this

File: Dockerfile (hunk 1 above)
Comment (arghyaiitb) [reply Dockerfile]: fixed

**Review by dmonark — APPROVED** [review]

**Pull request merged**